e2e_samples/parking_sensors_synapse/synapse/workspace/integrationRuntime/AutoResolveIntegrationRuntime.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Remove the auto-generated synapse files for auto resolve integration runtime, etc.